devops/staging/values.yaml

```diff
@@ -98,7 +101,7 @@ iam-cache-server-helm:
     DOMAIN_NOTIFIER_ADDRESS: 0xeea658026d6CDede4380D3aD030beAC911758A93
     RESOLVER_V1_ADDRESS: 0xf5EA22c1F799d425356c2aab2004200Ab4490D2b
     RESOLVER_V2_ADDRESS: 0xcf72f16Ab886776232bea2fcf3689761a0b74EfE
-    ASSETS_MANAGER_ADDRESS: 0x84d0c7284A869213CB047595d34d6044d9a7E14A
+    ASSETS_MANAGER_ADDRESS: 0x524563FeA0c9b54B337F0bAc366A9f541d26b3ea
```

Comment:
This should probably be checked as well. Is `0x524563FeA0c9b54B337F0bAc366A9f541d26b3ea` the right value?

Reply:
We should not change it, because assets was not migrated. I checked this by quering identity owners of several last assests